libs/core/src/lib/toggle/toggle.component.html

```diff
@@ -1,15 +1,25 @@
 <label fd-form-label [attr.for]="innerInputId">
-    <span class="fd-toggle fd-form__control" [ngClass]="(this.size ? ('fd-toggle--' + this.size) : '')">
+    <span class="fd-toggle"
+          [ngClass]="
+          (size ? ('fd-toggle--' + size) : '') +
+          (semantic ? ' fd-toggle--semantic' : ' ') +
```

Comment:
I think there should not be a space in between the quotes

Reply:
There should be quotes, because it's treaten as a string, without empty string it will be 
`fd-toggle--sizefd-toggle--semantic`

Reply:
quotes yes but there is a space on line 5 in the single quotes